packages-ts/starknet-gauntlet-multisig/package.json

```diff
@@ -25,7 +25,7 @@
     "bundle": "yarn build && pkg ."
   },
   "dependencies": {
-    "@chainlink-dev/starknet-multisig": "0.0.1",
+    "@chainlink-dev/starknet-multisig": "0.1.0",
```

Comment:
Lauri mentioned on Telegram they published the package: https://www.npmjs.com/package/starsign-multisig

But it's weird. The version published is v0.2.1, and the latest in repo is v1.0.0 (also different name): https://github.com/eqlabs/starknet-multisig/blob/main/contracts/package.json

Let's leave our package for now, and figure this out.